core/src/main/java/com/linecorp/armeria/server/ServiceRequestContextBuilder.java

```diff
@@ -82,6 +82,7 @@ public void serverStarting(Server server) {
     private RoutingResult routingResult;
     @Nullable
     private ProxiedAddresses proxiedAddresses;
+    private boolean useCancellationScheduler;
```

Comment:
This should go to the parent class and affect the behavior of both `{Client,Service}RequestContextBuilder`